zerver/lib/upload/__init__.py

```diff
@@ -53,8 +51,12 @@ def get_file_info(user_file: UploadedFile) -> Tuple[str, str]:
 
 # Common and wrappers
 if settings.LOCAL_UPLOADS_DIR is not None:
+    from zerver.lib.upload.local import LocalUploadBackend
+
     upload_backend: ZulipUploadBackend = LocalUploadBackend()
 else:
+    from zerver.lib.upload.s3 import S3UploadBackend
+
     upload_backend = S3UploadBackend()  # nocoverage
```

Comment:
I think this `nocoverage` comment is misplaced, and should have been on the `else`.

Reply:
It was pre-existing -- I just failed to move it.